packages/client-authentication/index.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+const debug = require('debug')('xmpp:client:authentication')
+
+function authenticate (creds, cb) {
+  const auth = this.authenticators.find(auth => auth.match(this.features))
```

Comment:
we're not dropping support for 0.12 (nor most browsers :) )

It's still to be done but I'll setup babel